server/auth/sso/sso.go

```diff
@@ -179,7 +188,7 @@ func newSso(
 	if err != nil {
 		return nil, fmt.Errorf("failed to create JWT encrpytor: %w", err)
 	}
-	log.WithFields(log.Fields{"redirectUrl": config.RedirectURL, "issuer": c.Issuer, "clientId": c.ClientID, "scopes": config.Scopes}).Info("SSO configuration")
+	log.WithFields(log.Fields{"redirectUrl": config.RedirectURL, "issuer": c.Issuer, "issuerAlias": c.IssuerAlias, "clientId": c.ClientID, "scopes": config.Scopes}).Info("SSO configuration")
```

Comment:
since `c.IssuerAlias` is optional, can we log that when it's enabled or something meaningful when it's not enabled?